pkg/system/phase2_creating.go

```diff
@@ -519,6 +529,51 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &bucketLogVolumeMounts)
 			}
+
+			if r.NooBaa.Spec.NotificationsPVC != nil {
+				notificationVolumeMounts := []corev1.VolumeMount{{
+					Name:      "notif-vol",
+					MountPath: "/var/logs/notifications",
+				}}
+				util.MergeVolumeMountList(&c.VolumeMounts, &notificationVolumeMounts)
+
+				notificationVolumes := []corev1.Volume {{
+					Name: "notif-vol",
+					VolumeSource: corev1.VolumeSource {
+						PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource {
+							ClaimName: *r.NooBaa.Spec.NotificationsPVC,
+						},
+					},
+				}}
+				util.MergeVolumeList(&podSpec.Volumes, &notificationVolumes)
+
+				//find secrets that tell us how to connect to remote notifications servers,
+				//mount them so core can read them
+				notificatoinSecrets := &corev1.SecretList{}
+				noobaaNotifSelector, _ := labels.Parse("app=noobaa,noobaa=notifications")
+				util.KubeList(notificatoinSecrets, &client.ListOptions{Namespace: options.Namespace, LabelSelector: noobaaNotifSelector})
```

Comment:
What happens if we don't find any secret? Do we have a default? we should reject? warn?

Reply:
Using default doesn't make sense, because user at least needs to specify url for remote server and its protocol (http/kafka).
I can look into going into reject, but that can be done in operator only if there are no secrets at all. It would still be possible to be missing some of the secrets and operator won't have a way to know that.

Reply:
I think we should give the user as much info as we can. If we don't have any secrets and we know the feature won't work this way - we should at least write a warning in the logs (I think we should also Reject)
It can be nice to also check that the secret has the needed fields we expect and maybe add some documentation about how the secret is supposed to look.

Reply:
maybe the user should give us the specific secret that he wants us to use and we won't just list all the secrets? WDYT?